examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
+
+## Tutorial
+
+Steps
+1. create IPFS instance
+
+First, we need an IPFS instance that we'll use throughout the tutorial. For the sake of convinience, we'll use `ipfs-daemon` module to create and IPFS instance.
```

Comment:
I'll rework this in a way that the init dance is in its own file and we include `js-ipfs` instead of `ipfs-daemon`. 

I don't think we'll get to merge ipfs-daemon to js-ipfs within two weeks given how massive public API change that is. Let me know if you think we can.